packages/prometheus/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.15.3"
```

```suggestion
- version: "1.16.0"
```
I think it should be next minor version, because the `type: enhancement` (not `type: bugfix`)